microsoft-365/compliance/sensitivity-labels-teams-groups-sites.md

```diff
@@ -109,18 +108,64 @@ After you enable the preview, use the following steps to create or edit sensitiv
 > [!IMPORTANT]
 > Only the site and group settings take effect when you apply a label to a team, group, or site. Other settings, such as encryption and content marking, aren't applied to all content within the team, group, or site. Similarly, if you create a label and don't turn on site and group settings, the label will still be available when users create teams, groups, and sites, but it won't do anything when users apply it.
 
-[Learn how to publish a sensitivity label](/microsoft-365/compliance/sensitivity-labels#what-label-policies-can-do)
+> [!IMPORTANT]
+> Only the site and group settings take effect when you apply a label to a team, group, or site. Other settings, such as encryption and content marking, aren't applied to all content within the team, group, or site. Similarly, if you create a label and don't turn on site and group settings, the label will still be available when users create teams, groups, and sites, but it won't do anything when users apply it.
```

Comment:
@cabailey  I would slightly reword "but it won't do anything when users apply it."
to 
"but it will classify without applying any settings."